build/azure-pipelines/win32/sql-product-build-win32.yml

```diff
@@ -51,13 +51,6 @@ steps:
       git merge $(node -p "require('./package.json').distro")
     displayName: Merge distro
 
-  - powershell: |
```

Comment:
Because the mixin-quality brings in the files I need (that I have added in ads-release) based on the quality that we specify, instead of merging a specific commit